### PR TITLE
Raise `IModelConnection.onChange` when a new `BlankConnection` is created

### PR DESCRIPTION
**Pull request by grigasp**
Fixes https://github.com/iTwin/itwinjs-core/issues/7498

### CODE REVIEW

**Review by saskliutas — APPROVED** [review]

File: core/frontend/src/test/BlankConnection.test.ts [hunk 1]
```diff
@@ -1,10 +1,11 @@
 /*---------------------------------------------------------------------------------------------
-* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
-* See LICENSE.md in the project root for license terms and full copyright notice.
-*--------------------------------------------------------------------------------------------*/
-import { afterAll, beforeAll, describe, expect, it } from "vitest";
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
```
Comment (MartynasStrazdas) [inline core/frontend/src/test/BlankConnection.test.ts:4]: Formatting.

**Review by MartynasStrazdas — APPROVED** [review]

File: core/frontend/src/test/BlankConnection.test.ts (hunk 1 above)
Comment (grigasp) [reply core/frontend/src/test/BlankConnection.test.ts:4]: what's wrong with it?

Comment (MartynasStrazdas) [reply core/frontend/src/test/BlankConnection.test.ts:4]: It seems there are unnecessary spaces.

File: core/frontend/src/test/BlankConnection.test.ts [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
+import { afterAll, beforeAll, describe, expect, it, vi } from "vitest";
```
Comment (JonasDov) [inline core/frontend/src/test/BlankConnection.test.ts:5]: I think we should keep a space between copyright comment and first import statement.

```suggestion
 *--------------------------------------------------------------------------------------------*/
 
import { afterAll, beforeAll, describe, expect, it, vi } from "vitest";
```

**Review by JonasDov — APPROVED** [review]

File: core/frontend/src/test/BlankConnection.test.ts (hunk 2 above)
Comment (grigasp) [reply core/frontend/src/test/BlankConnection.test.ts:5]: Core packages don't have this newline in their source files, so I'd rather not add it for consistency.

**Review by pmconne — APPROVED** [review]

**Pull request merged**